sys/zuse.hoon

```diff
@@ -7185,6 +7198,77 @@
   =,  mimes:html
   =,  ethe
   |%
+  ++  sign-transaction
```

Comment:
Are there test vectors for signature transactions that could be added to the test suite?

Reply:
I haven't come across any official test data, but I'll definitely add in tests against results provided by web3.js